configs/ajio_local_ios_config.properties

```diff
@@ -0,0 +1,20 @@
+RUNNER=distribute
+FRAMEWORK=cucumber
+RUNNER_LEVEL=methods
+CAPS=./caps/ajio_local_ios_capabilities.json
+APPLITOOLS_CONFIGURATION=./configs/applitools_config.json
+ENVIRONMENT_CONFIG_FILE=./src/test/resources/environments.json
+IS_VISUAL=false
+LOG_DIR=target
+LOG_PROPERTIES_FILE=./src/test/resources/log4j.properties
+PARALLEL=1
+PLATFORM=iOS
+PROXY_KEY=HTTP_PROXY
+REPORT_PORTAL_FILE=src/test/resources/reportportal.properties
+RUN_IN_CI=false
+TARGET_ENVIRONMENT=prod
+LAUNCH_NAME_SUFFIX= on 'prod' Environment
+TEST_DATA_FILE=./src/test/resources/testData.json
+CLEANUP_DEVICE_BEFORE_STARTING_EXECUTION=false
+BROWSER_CONFIG_FILE=./configs/browser_config.json
+APPIUM_JS_PATH=/Users/mayank17.gupta/.appium/node_modules/appium/lib/main.js
```

Comment:
This APPIUM_JS_PATH config is required as part of ATD, without this test case will not run. Shall I remove my username and enter "Your system UserName" and then push the change. Please suggest.

This is same as "ajio_local_config.properties" I have created a separate copy to avoid confusion the local properties is capable of running both Android and iOS tests. However in order to avoid confusion for iOS setup i created this copy.